packages/tasks/src/model-libraries.ts

```diff
@@ -469,6 +469,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml"`,
 	},
+	open_oasis: {
```

Comment:
```suggestion
	open-oasis: {
```

(for naming convention, even though it's not followed by every library, typically `open_clip`)